README.md

```diff
@@ -1 +1,194 @@
-# Radium Grid Component
+# Radium Grid
+
+Radium Grid is a powerful, no-fuss grid system component for React. It combines the best decisions from the ecosystem of (S)CSS-based grid systems and implements them hack-free on top of Flexbox.
+
+## What makes Radium Grid special?
+- Declarative layout using JSX.
+- Sensible defaults.
+- Uses arbitrary fractions for cell widths. No more 12-column straitjacket!
+- Infers rows from the given cell sizes. No need for explicit rows or extra `<div>`s!
+- Customizable cell alignment, including a hack-free vertical align!
+- Customizable fixed-width and fluid-width gutters. Just pass a CSS unit!
+- Customizable media queries for breakpoint definitions.
+- Uses Radium Style to handle breakpoint changes efficiently.
+- Accepts `style` arrays and resolves them with Radium Style.
+- Cascading defaults: set cell props on:
+  - all cells for all sizes,
+  - all cells for individual sizes,
+  - a single cell for all sizes,
+  - a single cell for individual sizes,
+  - ...with the lowest props in the tree overriding parent props.
+
+## Prerelease
+Although this is prerelease software, we don't anticipate breaking public API changes.
+
+## Installation
+`npm install --save radium-grid`
+
+## Usage
+```es6
+import { Grid, Cell } from 'radium-grid';
+
+const App = () => {
+  return (
+    <Grid>
+      <Cell>
```

Comment:
`Grid` and `Cell`, or `Grid` and `Grid.Cell` ?

`Grid.Cell` has the nice benefit of making the relationship more clear, but it's more verbose.

I have no strong feelings.